src/components/NewBookForm.js

```diff
@@ -47,31 +43,23 @@ const AddForm = () => {
   };
 
   return (
-    <form onSubmit={handleAdd}>
-      <label htmlFor="bookTitle">
-        Title
-        <input type="text" id="bookTitle" placeholder="Title" value={title} onChange={handleTitleChange} />
-      </label>
-      <br />
-      <label htmlFor="bookAuthor">
-        Author
-        <input type="text" id="bookAuthor" placeholder="Author" value={author} onChange={handleAuthorChange} />
-      </label>
-      <br />
-      <label htmlFor="bookCategory">
-        Category
-        <select id="bookCategory" value={category} onChange={handleCategoryChange}>
-          {categories.map((category) => (
-            <option key={category} value={category}>
-              {category}
-            </option>
-          ))}
-        </select>
-      </label>
-      <br />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="form-container">
+      <h2 className="form-title">
+        ADD A NEW BOOK
+      </h2>
+      <form className="add-form" onSubmit={handleAdd}>
+        <label className="grow-3" htmlFor="bookTitle">
+          <input className="input-control input-title" type="text" id="bookTitle" placeholder="Title" value={title} onChange={handleTitleChange} required />
+        </label>
+        <br />
```

Comment:
[OPTIONAL]
Please DO NOT use `<br />` tag for adding some space because it's considered as Bad Practice. Industry’s standards encourage developers to use margins to add a space between elements because it's considered as Best practice. 👍

Reply:
Oh is a bad habit, you are right...